### PR TITLE
Add new committers

### PR DESCRIPTION
**Pull request by EnricoMi**

### CODE REVIEW

**Comment by EnricoMi** [comment]
There are 9 more current committers that are missing from that list. Shall I add those too? I lack their organizations, though.

**Comment by roryqi** [comment]
> There are 9 more current committers that are missing from that list. Shall I add those too? I lack their organizations, though.

Yes, you can. I can tell you their organization.

**Comment by EnricoMi** [comment]
> > There are 9 more current committers that are missing from that list. Shall I add those too? I lack their organizations, though.
> 
> Yes, you can. I can tell you their organization.

Great, you can suggest them in the changes tab.

**Comment by EnricoMi** [comment]
Shall we sort the names alphabetically?

File: community/project committers.md [hunk 1]
```diff
@@ -36,3 +36,12 @@
   | Chenyu Zheng    | Ke Holdings  |
   | Qing Liu        | Lalamove     |
   | Enrico Minack   | G-Research   |
+  | Colin           |              |
```
Comment (jerqi) [inline community/project committers.md:39]: Colin is at Ebay

File: community/project committers.md [hunk 2]
```diff
@@ -36,3 +36,12 @@
...
+  | Felix Cheung    |              |
+  | Junping Du      |              |
```
Comment (jerqi) [inline community/project committers.md:41]: Datastrato

File: community/project committers.md [hunk 3]
```diff
@@ -36,3 +36,12 @@
...
+  | Liu Xun         |              |
```
Comment (jerqi) [inline community/project committers.md:42]: Liu Xun is at Datastrato

File: community/project committers.md [hunk 4]
```diff
@@ -36,3 +36,12 @@
...
+  | Ricky Ma        |              |
```
Comment (jerqi) [inline community/project committers.md:43]: Ricky is at Tencent.

File: community/project committers.md [hunk 5]
```diff
@@ -36,3 +36,12 @@
...
+  | He Qi           |              |
```
Comment (jerqi) [inline community/project committers.md:44]: It's me. This is duplicated.

File: community/project committers.md [hunk 6]
```diff
@@ -36,3 +36,12 @@
...
+  | Weiwei Yang     |              |
```
Comment (jerqi) [inline community/project committers.md:45]: Weiwei is at Apple.

File: community/project committers.md [hunk 7]
```diff
@@ -36,3 +36,12 @@
...
+  | Yang Jie        |              |
```
Comment (jerqi) [inline community/project committers.md:46]: Yang Jie is at Baidu.

File: community/project committers.md [hunk 8]
```diff
@@ -36,3 +36,12 @@
...
+  | Zhankun Tang    |              |
```
Comment (jerqi) [inline community/project committers.md:47]: Zhankun Tang is at Cloudera.

File: community/project committers.md [hunk 9]
```diff
@@ -36,3 +36,12 @@
   | Chenyu Zheng    | Ke Holdings  |
   | Qing Liu        | Lalamove     |
   | Enrico Minack   | G-Research   |
+  | Colin           |              |
+  | Felix Cheung    |              |
```
Comment (jerqi) [inline community/project committers.md:40]: Felix is at NVIDIA

File: community/project committers.md [hunk 10]
```diff
@@ -35,3 +35,13 @@
   | Xianming Lei    | DiDi Global  |
   | Chenyu Zheng    | Ke Holdings  |
   | Qing Liu        | Lalamove     |
+  | Enrico Minack   | G-Research   |
+  | Colin           |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Colin           |     Ebay     |
```

File: community/project committers.md [hunk 11]
```diff
@@ -35,3 +35,13 @@
...
+  | Felix Cheung    |              |
```
Comment (jerqi) [inline community/project committers.md:40]:
```suggestion
  | Felix Cheung    |  NVIDIA |
```

Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Felix Cheung    |    NVIDIA    |
```

File: community/project committers.md [hunk 12]
```diff
@@ -35,3 +35,13 @@
...
+  | Junping Du      |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Junping Du      |  Datastrato  |
```

File: community/project committers.md [hunk 13]
```diff
@@ -35,3 +35,13 @@
...
+  | Liu Xun         |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Liu Xun         |  Datastrato  |
```

File: community/project committers.md [hunk 14]
```diff
@@ -35,3 +35,13 @@
...
+  | Ricky Ma        |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Ricky Ma        |   Tencent    |
```

File: community/project committers.md [hunk 15]
```diff
@@ -35,3 +35,13 @@
...
+  | He Qi           |              |
```
Comment (roryqi) [inline community/project committers.md]: This is duplicated.

File: community/project committers.md [hunk 16]
```diff
@@ -35,3 +35,13 @@
...
+  | Weiwei Yang     |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Weiwei Yang     |    Apple     |
```

File: community/project committers.md [hunk 17]
```diff
@@ -35,3 +35,13 @@
...
+  | Yang Jie        |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Yang Jie        |     Baidu    |
```

File: community/project committers.md [hunk 18]
```diff
@@ -35,3 +35,13 @@
...
+  | Zhankun Tang    |              |
```
Comment (roryqi) [inline community/project committers.md]:
```suggestion
  | Zhankun Tang    |   Cloudera   |
```

File: community/project committers.md (hunk 15 above)
Comment (roryqi) [inline community/project committers.md]:
```suggestion
```

**Comment by roryqi** [comment]
> Shall we sort the names alphabetically?

It's ok.

**Comment by roryqi** [comment]
Jerry Shao's organization should be `Datastrato` instead of `DataStrato`. He Qi's organization should be `Datastrato`.

File: community/project committers.md [hunk 19]
```diff
@@ -35,3 +35,12 @@
   | Xianming Lei    | DiDi Global  |
   | Chenyu Zheng    | Ke Holdings  |
   | Qing Liu        | Lalamove     |
+  | Enrico Minack   | G-Research   |
+  | Colin           | eBay         |
```
Comment (roryqi) [inline community/project committers.md]: Colin is Junjie Ma.

Comment (EnricoMi) [reply community/project committers.md]: removed

**Review by roryqi — APPROVED** [review]
LGTM.

File: community/project committers.md [hunk 20]
```diff
@@ -35,3 +35,11 @@
   | Xianming Lei    | DiDi Global  |
   | Chenyu Zheng    | Ke Holdings  |
   | Qing Liu        | Lalamove     |
+  | Enrico Minack   | G-Research   |
+  | Felix Cheung    | NVIDIA       |
+  | Junping Du      | Datastrato   |
+  | Liu Xun         | Datastrato   |
+  | Ricky Ma        | Tencent      |
+  | Weiwei Yang     | Apple        |
+  | Yang Jie        | Baidu        |
```
Comment (roryqi) [inline community/project committers.md]: `Yang Jie` and `Jie Yang`is duplicated, too.

Comment (EnricoMi) [reply community/project committers.md]: Yang Jie is to be removed? Jie Yang is correct?

Comment (roryqi) [reply community/project committers.md]: Yes, Jie Yang is correct.

Comment (EnricoMi) [reply community/project committers.md]: done

**Pull request merged**